adapta/storage/models/format.py

```diff
@@ -49,20 +55,20 @@ def deserialize(self, data: bytes) -> T:
         """
 
 
-class DataFrameParquetSerializationFormat(SerializationFormat[DataFrame]):
+class DataFrameParquetSerializationFormat(SerializationFormat[PandasDataFrame]):
```

Comment:
I would call this something with Pandas. If they update, it will break, but it is a very small and easy fix

Reply:
I'm fine with this.

However, it is breaking and according to semver requires a bump of major version. Are we ok with this?

Reply:
Personally, I would just bump the minor. It also depends on how strictly we interpret the semver guideline. In our other projects, our interpretation is not that strict.

I know @george-zubrienko sometimes argue that if the behavior is fully unchanged, and you are just renaming something, it is okay to just bump the minor.

Reply:
@george-zubrienko, ok will change accordingly.

Matthias is suggesting to rename the class itself, such that it explicitly indicates that it operates on Pandas DFs. This would be breaking

Reply:
If you rename the class, then do it other way: add an alias for this one called PandasDataframeSerializer. Then we can break it in later update.

Reply:
Aliases added